DBbot/classes/column.ts

```diff
@@ -58,4 +58,87 @@ export class Column {
     public addOperator(operator: Operator): void {
         this.operatorsArray.push(operator);
     }
+
+    public mean(): number {
+        if (this.dataType === "string") {
+            throw new Error("Cannot calculate mean for string data type");
+        }
+        const sum = this.rows.reduce((acc, curr) => {
+            if (isNaN(curr)) return acc + 0;
+            return acc + curr;
+        }, 0);
+        const mean = sum / this.rows.length;
+        return Math.round(mean * 100) / 100;
+    }
+
+    public mode(): number {
+        if (this.dataType === "string") {
+            throw new Error("Cannot calculate mode for string data type");
+        }
+        const counts = this.rows.reduce((acc, curr) => {
```

Comment:
What is acc and curr? try to not use shortcuts in variable names

Reply:
fixed